src/app/item-popup/WeaponDeepsightInfo.tsx

```diff
@@ -9,18 +14,41 @@ export function WeaponDeepsightInfo({ item }: { item: DimItem }) {
   const record = item.patternUnlockRecord;
   const relevantObjectives = record?.objectives.filter((o) => !o.complete);
 
+  const defs = useD2Definitions()!;
+  const harmonizerIcon = defs.InventoryItem.get(DEEPSIGHT_HARMONIZER).displayProperties.icon;
```

Comment:
Add `?.` for when this definition inevitably gets removed from the defs. You'll need to handle the undefined case too.

```suggestion
  const harmonizerIcon = defs.InventoryItem.get(DEEPSIGHT_HARMONIZER)?.displayProperties.icon;
```